lambda-loader/src/data.py

```diff
@@ -68,7 +72,9 @@ def __append_rating_to_list(self,rating,item):
         if item['Ratings'] and item['Ratings'][-1] == rating: # List is not empty and No updates to ratings list, return.
             return item 
 
-        item['Ratings'].append(rating) 
+        item['Ratings'].append(rating)
+
+        print(f"{item} was appended on")
```

Comment:
Does this work? Do you need to use `json.dumps()` to convert an obj->str?

Reply:
Looks like it works 

> {'Region': 'US', 'PlayerName': 'diyingli', 'TTL': 1616223600, 'Ratings': [14720]} was appended on